doc/faq.md

```diff
@@ -176,6 +176,29 @@ re-install Convos:
 
 Please [contact us](/doc/#get-in-touch) if the above instructions do not work.
 
+## Why convos does not restart after an upgrade of convos code?
+
+Some changes in the convos code do not require any special operation (for instance if you modify only assets and/or themes). 
+
+If you update the convos code and get stuck to the loading page after a refresh like this :
+
+<img height="300px" src="../public/screenshots/2020-05-15-09-40-05-reload-error.png" alt="Stuck during reload">
+
+Then you likely have pulled changes that require you to do a service restart. 
+
+Then simply execute :
+
+```
+$ killall convos    
```

Comment:
Maybe this needs an alternative like `kill $PID_OF_RUNNING_CONVOS`?

Reply:
I added this alternative, is it ok for you ?